pennylane/pauli/dla/lie_closure.py

```diff
@@ -134,22 +135,29 @@ def lie_closure(
     while (new_length > old_length) and (epoch < max_iterations):
         if verbose:
             print(f"epoch {epoch+1} of lie_closure, DLA size is {new_length}")
+
         for ps1, ps2 in itertools.combinations(vspace.basis, 2):
             com = ps1.commutator(ps2)
+            com.simplify()
```

Comment:
Any concerns about performance degradation by adding a simplification?

Reply:
Curiously I dont actually see any regressions in some example timings (would have expected at least some minor slow down). But either wary, that work is necessary if we want to be able to treat pauli _sentences_ as well, anyway.